test/gtest/uct/ib/test_ud.cc

```diff
@@ -918,6 +918,91 @@ UCS_TEST_SKIP_COND_P(test_ud, ctls_loss,
 
 UCT_INSTANTIATE_UD_TEST_CASE(test_ud)
 
+
+class test_ud_peer_failure : public ud_base_test {
```

Comment:
is this a merge artifact? if not why needed to change UD tests?

Reply:
merge artifact; will fix.